### PR TITLE
Fix/name change

### PR DESCRIPTION
**Pull request by JacOng17**
Removed tags on Project & tool cards & changed text on search results.
image

### CODE REVIEW

File: _products_projects/component-library.md [hunk 1]
```diff
@@ -7,7 +7,6 @@ excerpt: Resource of common application components for developers
 image: /assets/img/projects/
 image_accessibility:
 image_icon:
-tag: Component Library
 expiration_date:
 project_url: "[]()"
```
Comment (Br3nda) [inline _products_projects/component-library.md]: possibly these can be removed too if they're not used.

Comment (JacOng17) [reply _products_projects/component-library.md]: It's removed already in this PR :stuck_out_tongue:

Comment (Br3nda) [reply _products_projects/component-library.md]: Github disagrees. you ave not removed line 12 in this PR.

Comment (Br3nda) [reply _products_projects/component-library.md]: these:
``` project_url: "[]()" ```

Comment (JacOng17) [reply _products_projects/component-library.md]: image

Comment (Br3nda) [reply _products_projects/component-library.md]: image
Still there!

Comment (JacOng17) [reply _products_projects/component-library.md]: Not too sure why you're seeing something different on your end. This screenshot shows what I'm seeing.

Comment (Br3nda) [reply _products_projects/component-library.md]: And i have put big pick arrows at that line on your screen shot... It's still there.....

:confused:

Comment (JacOng17) [reply _products_projects/component-library.md]: Ah, as in you meant to say remove these:
image

I though you meant remove the tag.

Comment (JacOng17) [reply _products_projects/component-library.md]: I'll remove them in a new pr

Comment (Br3nda) [reply _products_projects/component-library.md]: nope. comment is on line 12, which has this content: ` project_url: "[]()"` and i've marked in your screenshot.

Comment (JacOng17) [reply _products_projects/component-library.md]: Ok, I understand now :+1: I'm actually gonna remove the ones not in use too cos the Tools card don't need some fields.

**Pull request merged**